### ISSUE TITLE
Restrict Mode

### ISSUE THREAD

**Issue by MSZ-MGS** [yuliskov/SmartTube#2463]
I hope to have restrict mode option which basically filter adult contents on YouTube.
This option is available on the YouTube app. 

Regards...

**Comment by bkarakashev** [comment]
I am also interested in having a restricted mode, something like YouTube Kids. I have a separated account for my kid but I can still see some odd content and maybe some news or random stuff. I would like to be able to only see kids friendly content, Thank you.

**Comment by techgamers2** [comment]
yes please add restricted mode ... i know child mode exist but it completely disables search
